starwars/src/components/StarWarsData.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import './StarWars.css';
+
```

Comment:
remember not to use class when labeling, this returns an error in your console stating invalid DOM property, use className